e2e/TestConstants.ts

```diff
@@ -85,13 +85,47 @@ export const TestConstants = {
     TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME: process.env.TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME || 'petclinic-dev-environment',
 
     /**
-     * Username used to log in MultiUser Che
+     * Name of namespace created for 'Install Che' on OCP 4.x by OperatorHub UI.
      */
-    TS_SELENIUM_USERNAME: process.env.TS_SELENIUM_USERNAME || 'che',
+    TS_INSTALL_CHE_PROJECT_NAME: process.env.TS_INSTALL_CHE_PROJECT_NAME || 'test-che-operator',
 
     /**
-     * Password used to log in MultiUser Che
+     * Username used to log in MultiUser Che.
      */
-    TS_SELENIUM_PASSWORD: process.env.TS_SELENIUM_PASSWORD || ''
+    TS_SELENIUM_USERNAME: process.env.TS_SELENIUM_CHE_USERNAME || '',
```

Comment:
There was `che` user name by default.

Reply:
@Katka92 please take a look, this changes may influence to your work